# HG changeset patch
# User Rik <rik@octave.org>
# Date 1380301038 25200
#      Fri Sep 27 09:57:18 2013 -0700
# Node ID 3dcd7fb9945d0cf28f015df5be9b4f51b82ba5ce
# Parent  8b692d9ea54ea0360f91fbdb34582c4a31bfe454
NEWS: Update with changes made to plot directory functions.

* NEWS: Update with changes made to plot directory functions.

diff --git a/NEWS b/NEWS
--- a/NEWS
+++ b/NEWS
@@ -139,63 +139,74 @@ Summary of important user-visible change
     replaced the two characters with the sequence '\', 'n'.
 
  ** A TeX parser has been implemented for the FLTK toolkit and is the default
     for any text object including titles and axis labels.  The TeX parser is
     supported only for display on a monitor, not for printing.
     A quick summary of features:
 
     Code         Feature     Example             Comment
+    -----------------------------------------------------------------
     _            subscript   H_2O                formula for water
     ^            exponent    y=x^2               formula for parabola
     \char        symbol      \beta               Greek symbol beta
     \fontname    font        \fontname{Arial}    set Arial font
     \fontsize    fontsize    \fontsize{16}       set fontsize 16
     \color[rgb]  fontcolor   \color[rgb]{1 0 1}  set magenta color 
     \bf          bold        \bfBold Text        bold font
     \it          italic      \itItalic Text      italic font
     \sl          slanted     \slOblique Text     slanted font
     \rm          normal      \bfBold\rmNormal    normal font
     {}           group       {\bf Bold}Normal    group objects
                              e^{i*\pi} = -1      complex example
 
+ ** The m-files in the plot directory have been overhauled.
+
+    The plot functions now produce output that is nearly visually compatible
+    with Matlab.  Plot performance has also increased, dramatically for some
+    functions such as comet and waitbar.  Finally, the documentation for most
+    functions has been updated so it should be clearer both how to use a
+    function and when a function is appropriate.
+
  ** The m-files in the image directory have been overhauled.
 
     The principal benefit is that Octave will now no longer automatically
     convert images stored with integers to doubles.  Storing images as uint8
     or uint16 requires only 1/8 or 1/4 the memory of an image stored using
     doubles.  For certain operations, such as fft2, the image must still be
     converted to double in order to work.
 
     Other changes include fixes to the way indexed images are read from a
     colormap depending on the image class (integer images have a -1 offset to
     the colormap row number).
 
- ** The imread and imwrite functions have been completely rewritten
+ ** The imread and imwrite functions have been completely rewritten.
 
     The main changes relate to the alpha channel, support for reading and
     writing of floating point images, implemented writing of indexed images,
     and appending images to multipage image files.
 
     The issues that may arise due to backwards incompatibility are:
 
       * imwrite no longer interprets a length of 2 or 4 in the third dimension
         as grayscale or RGB with alpha channel (a length of 4 will be saved
-        as CMYK image).  Alpha channel must be passed as separate argument.
+        as a CMYK image).  Alpha channel must be passed as separate argument.
+
       * imread will always return the colormap indexes when reading an indexed
         image, even if the colormap is not requested as output.
-      * transparency values are now inverted from the previous Octave versions
+
+      * transparency values are now inverted from previous Octave versions
         (0 is for completely transparent instead of completely opaque).
 
     In addition, the function imformats has been implemented to expand
     reading and writing of images of different formats through imread
     and imwrite.
 
- ** The colormap function now provides new options---"list", "register",
-    and "unregister"---to list all available colormap functions, and to
+ ** The colormap function now provides new options--"list", "register",
+    and "unregister"--to list all available colormap functions, and to
     add or remove a function name from the list of known colormap
     functions.  Packages that implement extra colormaps should use these
     commands with PKG_ADD and PKG_DEL statements.
 
  ** strsplit has been modified to be compatible with Matlab.  There
     are two instances where backward compatibility is broken.
 
     (1) Delimiters are now string vectors, not scalars.
@@ -243,18 +254,18 @@ Summary of important user-visible change
     In addition two new error functions erfi (imaginary error function) and
     dawson (scaled imaginary error function) have been added.
 
  ** The glpk function has been modified to reflect changes in the GLPK
     library.  The "round" and "itcnt" options have been removed.  The
     "relax" option has been replaced by the "rtest" option.  The numeric
     values of error codes and of some options have also changed.
 
- ** The default name of the Octave crash dump file is now called
-    octave-workspace instead of octave-core.
+ ** The default name of the Octave crash dump file is now 
+    "octave-workspace" instead of "octave-core".
 
  ** A citation command has been added to display information on how to
     cite Octave and packages in publications.  The package system will
     look for and install CITATION files from packages.
 
  ** The java package from Octave Forge is now part of core Octave.  The
     following new functions are available for interacting with Java
     directly from Octave:
